src/frame/js/components/Action.jsx

```diff
@@ -195,12 +195,12 @@ export class ActionComponent extends Component {
     }
 }
 
-export default connect(({app, ui: {text}, user}) => {
+export default connect(({ui: {text}, user, config}) => {
     return {
         user,
         actionPaymentCompletedText: text.actionPaymentCompleted,
-        integrations: app.integrations,
-        stripe: app.stripe
+        integrations: config.integrations,
+        stripe: config.stripe
```

Comment:
it was still part of the config call when I implemented it... I'll fix this once the backend is more "official".. most of the component changes are just to be able to render the widget with the new changes